C4SGSvc/src/main/java/org/c4sg/controller/UserProjectController.java

```diff
@@ -0,0 +1,44 @@
+package org.c4sg.controller;
+
+import org.c4sg.dto.ProjectDto;
+import org.c4sg.dto.UserProjectDto;
+import org.c4sg.exception.UserProjectException;
+import org.c4sg.service.UserProjectService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@CrossOrigin
+@RestController
+@RequestMapping("/api/userProjects")
+public class UserProjectController {
+
+    @Autowired
+    private UserProjectService userProjectService;
+
+    @CrossOrigin
+    @RequestMapping(value = "/users/{id}/projects/{projectId}", method = RequestMethod.POST)
```

Comment:
I know that we still don`t have authentication mechanism. Please mark this annotation with TODO: Replace explicit user{id} with AuthN user id.
But I meant we don`t need projectId because at time we post the operation the Id isn`t even generated.